redi-connect-backend/server/ssl-config.js

```diff
@@ -1,7 +1,7 @@
 var path = require('path')
 var fs = require('fs')
 
-if (!process.env.HTTP) {
+if (process.env.USE_HTTPS) {
```

Comment:
I think we missed changing that to `USE_HTTPS`. Can you please validate @ericbolikowski?

Reply:
Yes, correct. In my haste I only validated for USE_HTTPS being true. This is correct fix. Thanks.